rolling/aggregation/whole.go

```diff
@@ -0,0 +1,93 @@
+package aggregation
```

Comment:
Did not get if this file is completely new or has just been moved. 
Does it implement something new ?

Reply:
It has just been renamed from [rolling/aggregation/wholeaggregation.go](https://github.com/Metronlab/bow/pull/78/files#diff-3f745af336ea7f7c9cec51de52900773b942cbec1083584ffb8406ff6f81755f)
I found that the name was redundant with the path, so I made it shorter.
I have no idea why github is displaying the change like that...